cbas-libchart/Chart.yaml

```diff
@@ -0,0 +1,5 @@
+apiVersion: v2
+name: cbas-libchart
```

Comment:
On chart naming...

As we discussed earlier today, we're not sure exactly what will eventually end up in library charts. This one has two related components: C-BAS and the reverse proxy. I think we should to one of:
 * Rename this library chart to something like `terra-batch-libchart`
 * Split the above two components out into separate library charts

Reply:
I think I like option 1 for now. Let's briefly discuss naming today (we can keep the discussion time boxed to like 5 minutes). I have no objections to `terra-batch-libchart`